runtimes/elixir/elixir.md

```diff
@@ -0,0 +1,62 @@
+## Overview
+Currently, Clever Cloud supports Elixir based applications.
+
+## Create an Elixir application and add a PostreSQL database
```

Comment:
Why talk about a PostgreSQL database?

Reply:
As everything is really phoenix oriented (hi, env var names), I writted it assuming we were deplying a Phenix app, wich needs a PG database. WDYT ?

Reply:
Maybe you could mention that this is an example using a phoenix application and that there might be some nuances if the user isn't using phoenix (at least 10% of our elixir users run somehting else).

Reply:
thank you for the approach, will update when I'll have answer on the other comment then we'll be able to close

Reply:
✅